ledger/sandbox/BUILD.bazel

```diff
@@ -9,6 +9,70 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_image")
 load("@io_bazel_rules_docker//java:image.bzl", "java_image")
 load("@os_info//:os_info.bzl", "is_windows")
 
+compile_deps = [
+    "//daml-lf/archive:daml_lf_archive_reader",
+    "//daml-lf/archive:daml_lf_dev_archive_java_proto",
+    "//daml-lf/data",
+    "//daml-lf/engine",
+    "//daml-lf/interpreter",
+    "//daml-lf/language",
+    "//daml-lf/scenario-interpreter",
+    "//daml-lf/transaction",
+    "//daml-lf/transaction:transaction_java_proto",
+    "//daml-lf/transaction:value_java_proto",
+    "//language-support/scala/bindings",
+    "//ledger-api/rs-grpc-akka",
+    "//ledger-api/rs-grpc-bridge",
+    "//ledger-service/jwt",
+    "//ledger/ledger-api-akka",
+    "//ledger/ledger-api-auth",
+    "//ledger/ledger-api-client",
+    "//ledger/ledger-api-common",
+    "//ledger/ledger-api-domain",
+    "//ledger/ledger-api-health",
+    "//ledger/ledger-on-sql",
+    "//ledger/participant-state",
+    "//ledger/participant-state-index",
+    "//ledger/participant-state/kvutils",
+    "//libs-scala/contextualized-logging",
+    "//libs-scala/direct-execution-context",
+    "//libs-scala/resources",
+    "//libs-scala/resources-akka",
+    "@maven//:ch_qos_logback_logback_classic",
+    "@maven//:ch_qos_logback_logback_core",
+    "@maven//:com_auth0_java_jwt",
+    "@maven//:com_github_scopt_scopt_2_12",
+    "@maven//:com_google_guava_guava",
+    "@maven//:com_typesafe_akka_akka_actor_2_12",
+    "@maven//:com_typesafe_akka_akka_stream_2_12",
+    "@maven//:com_typesafe_config",
+    "@maven//:com_typesafe_play_anorm_2_12",
+    "@maven//:com_typesafe_play_anorm_tokenizer_2_12",
+    "@maven//:com_typesafe_scala_logging_scala_logging_2_12",
+    "@maven//:com_zaxxer_HikariCP",
+    "@maven//:io_dropwizard_metrics_metrics_core",
+    "@maven//:io_dropwizard_metrics_metrics_jmx",
+    "@maven//:io_grpc_grpc_netty",
+    "@maven//:io_grpc_grpc_services",
+    "@maven//:io_netty_netty_common",
+    "@maven//:io_netty_netty_handler",
+    "@maven//:io_netty_netty_transport",
+    "@maven//:io_zipkin_brave_brave",
+    "@maven//:net_logstash_logback_logstash_logback_encoder",
+    "@maven//:org_flywaydb_flyway_core",
+    "@maven//:org_scala_lang_modules_scala_java8_compat_2_12",
+    "@maven//:org_scalaz_scalaz_core_2_12",
+    "@maven//:org_slf4j_slf4j_api",
+]
+
+runtime_deps = [
```

Comment:
@aherrmann I believe you originally inlined the dependencies when introducing `buildozer`. This commits undoes it. Is that a problem?

Reply:
It's only for `//ledger/sandbox:ledger-api-server` and `//ledger/sandbox:sandbox`, which are actually the same thing but split so you can name the thing you depend on. I think we should probably make `//ledger/ledger-api-server` its own thing at some point.